plugins/generator-1.18.2/forge-1.18.2/templates/modbase/sounds.json.ftl

```diff
@@ -1,16 +1,13 @@
 {
 <#list sounds as sound>
 "${sound.getName()}": {
-  "category": "${generator.map(sound.getCategory(), "soundcategories", 1)}",
   <#if sound.getSubtitle()?has_content>"subtitle": "subtitles.${sound.getName()}",</#if>
   "sounds": [
     <#list sound.getFiles() as file>
     {
       "name": "${modid}:${file}",
       "stream": <#if sound.getCategory() == "record" || sound.getCategory() == "music">true<#else>false</#if>
-    }<#if file?has_next>,</#if>
-    </#list>
+    }<#sep>,</#list>
```

Comment:
I think the closing `<#list>` should still be on a separate line as it's the parent operator for `<#sep>`.